docs/DevGuide/OptionParsing.md

```diff
@@ -85,9 +85,10 @@ Example:
 The factory interface creates an object of type
 `std::unique_ptr<Base>` containing a pointer to some class derived
 from `Base`.  The list of creatable derived classes is specified in
-the `factory_classes` type alias in the metavariables, which should be
-a `tmpl::map` from base classes to lists of derived classes: \snippet
-Test_Factory.cpp factory_classes
+the `factory_creation` struct in the meatvariables, which must contain
```

Comment:
typo: meatvariables -> metavariables

Reply:
I kind of like the sound of Meatvariables though